### ISSUE TITLE
[3.1.190618.2-rc1] - Market whitelist - text overlap

### ISSUE THREAD

**Issue by allfory0u** [bitshares/bitshares-ui#2892]
**Describe the bug**
When you enter a symbol to add an asset to the market whitelist, the informational message overlaps the text.

**To Reproduce**
1. Go to [staging.bitshares.org/#/account/test/update-asset/BTS](https://staging.bitshares.org/#/account/test/update-asset/BTS)
2. Start typing the asset symbol
3. See how the message overlaps the text

**Expected behavior**
Don`t see how the message overlaps the text.

**Screenshots**
![create asset - white screen](https://user-images.githubusercontent.com/17462020/60667341-fa56db00-9e71-11e9-916c-34f9b1cc3662.jpg)

![QONRvCsecf](https://user-images.githubusercontent.com/17462020/60667350-017de900-9e72-11e9-8fb7-50ef35324968.gif)

**Desktop (please complete the following information):**
 - Windows 7
 - Chrome 71

**Comment by startailcoon** [comment]
This section should use the new style guide template to improve the user experience.

## Tasks
- Replace old components with new style-guide components

**Comment by sschiessl-bcp** [comment]
We should rather redesign the whole layout here, not sure how much sense that makes atm. 

@gibbsfromncis did redesign this, can you please point us to the issue? Also there is probably a mockup for the new wallet design for this?

**Comment by startailcoon** [comment]
@sschiessl-bcp true. We have a UX for this section that was prepared. UX was created as of #1604 and a code implementation was created by #2117 but was halted due to the pending wallet update.

My suggestion is to update this section with minor enhancements to remove the obvious issues, like overlapping text, with the new style-guide components. This would solve some of the current issues, while not taking to much effort.

Or we could update the whole section with the new UX created.
